### ISSUE TITLE
DATA

### ISSUE THREAD

**Issue by sotasky** [guokan987/HGCN#4]
Hi，Thank you for your share of Code!
I know the city data cannot open for the reason of copyright. Could you please present the random .pkl with data CORRECT shape. It is very useful for us to run the code! Thank you very much.

**Comment by guokan987** [comment]
> Hi，Thank you for your share of Code! I know the city data cannot open for the reason of copyright. Could you please present the random .pkl with data CORRECT shape. It is very useful for us to run the code! Thank you very much.

Please support your emails, I can give you the data of website by BaiDu Cloud disk!

**Comment by caisikai** [comment]
> > Hi，Thank you for your share of Code! I know the city data cannot open for the reason of copyright. Could you please present the random .pkl with data CORRECT shape. It is very useful for us to run the code! Thank you very much.
> 
> Please support your emails, I can give you the data of website by BaiDu Cloud disk!

My email is sikai.cai@qq.com. Thank you very much!

**Comment by ray-hr** [comment]
Thank you very much for your share of Code ! Your paper  has benefited me a lot . Could you share me the data set? Here is my emails :[ 2021232001@chd.edu.cn ](url) Thank you very much!

**Comment by xue-983** [comment]
Thank you very much for your share of Code ! Your paper has benefited me a lot . Could you share me the data set? Here is my emails :1968875837@qq.com Thank you very much!

**Comment by WangYibucea** [comment]
Hello, I have gone through the protocol of dropping Gaia data. However, I would like a copy of the cleaned data for a fair comparison. Thank you very much if you can provide it to me. Here is my email:1207918198@qq.com.

**Comment by guokan987** [comment]
> Hello, I have gone through the protocol of DIDI Gaia data. However, I would like a copy of the cleaned data for a fair comparison. Thank you very much if you can provide it to me. Here is my email:[1207918198@qq.com](mailto:1207918198@qq.com).

已发

**Comment by WangYibucea** [comment]
非常感谢。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "guokan987/HGCN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年4月14日(星期四) 中午11:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [guokan987/HGCN] DATA (Issue #4)





  
Hello, I have gone through the protocol of DIDI Gaia data. However, I would like a copy of the cleaned data for a fair comparison. Thank you very much if you can provide it to me. Here is my ***@***.***
  
已发

**Comment by wang-0311** [comment]
Thank you so much for sharing the code! I have benefited a lot from your paper. Can you share the processed dataset? I would like to reproduce your paper below. Here is my email: [1084610669@qq.com] Thanks a lot!

**Comment by weiyeah** [comment]
Hi, thank you very much for sharing the code, I also hope to get your datasets, so that I can run through the code to improve my personal learning. Here is my email: weiyeah@qq.com. Thanks a lot!

**Comment by guokan987** [comment]
> [weiyeah@qq.com](mailto:weiyeah@qq.com)

ok

**Comment by stormGa** [comment]
你好，可以分享一下处理过后的数据集吗？如果可以的话，非常感谢！我的邮箱为499536117@qq.com

**Comment by guxiaoyan5** [comment]
非常感谢您分享代码！我从你的论文中受益匪浅。你可以分享处理过的数据集吗？这是我的邮箱：[21120355@bjtu.edu.cn](mailto:21120355@bjtu.edu.cn) 非常感谢！

**Comment by EnergyYou** [comment]
非常感谢您分享代码，请问可以分享一下数据集吗？这是我的邮箱：1369668548@qq.com 非常感谢！

**Comment by isharrisleung** [comment]
Thanks for your excellent work, i'm vrey interesting in it ! I have researched some related issues, and I wonder if you can send me your processed dafaset. Here is my email: chris_lht@outlook.com. Thank you very much!

**Comment by mewtwov** [comment]
哈喽您好，最近也在学习相关方面的知识，十分感谢您的代码，尝试从滴滴官网获取数据集，但链接已经失效，如果可以的话，可以请您分享一下原始数据集吗？十分感谢！邮箱：mustvv@foxmail.com

盼复。

**Comment by ZHUGUODONG1** [comment]
Hello, thank you very much for sharing the code. I am currently doing research in this direction and I hope to get the dataset, but the relevant project has been closed and I cannot apply for it. Can you send the dataset to my email address? This is my email: 2104210036@cugb.edu.cn. Thank you!

**Comment by KHMalek** [comment]
Hi  @guokan987 , is it possible to share the Didi Chuxing GAIA dataset? I was wondering if it would be possible for you to share the dataset for educational and research purposes. This is my  email : melek.khemiri@ensi-uma.tn

Thank you for your help.

**Comment by guokan987** [comment]
> Hi @guokan987 , is it possible to share the Didi Chuxing GAIA dataset? I was wondering if it would be possible for you to share the dataset for educational and research purposes. This is my email : [melek.khemiri@ensi-uma.tn](mailto:melek.khemiri@ensi-uma.tn)
> 
> Thank you for your help.

Yes, you can only apply it to education and research, and cannot disseminate this data online. Thank you for your support!

**Comment by learner0102** [comment]
Hello, thank you very much for sharing the code, the paper has benefited me a lot, can you please share your dataset, thank you very much! Here is my email address : 983583804@qq.com. Thanks a lot!

**Comment by wshuo123** [comment]
@guokan987 您好，非常感谢您分享的代码，请问一下可以分享滴滴GAIA的原始数据集吗，我目前是一名在读研二学生，最近的课题需要用到这个数据，保证数据只用于科研用途。我的邮箱是1935786075@qq.com
